diff --git a/node_modules/react-quill/lib/component.js b/node_modules/react-quill/lib/component.js
index 9e92461..1b52d43 100644
--- a/node_modules/react-quill/lib/component.js
+++ b/node_modules/react-quill/lib/component.js
@@ -282,7 +282,7 @@
 	},
 
 	getEditorNode: function() {
-		return ReactDOM.findDOMNode(this.editor);
+		return this.editor;
 	},
 
 	/*
diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 1618195..81f6920 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -370,7 +370,7 @@
   },
 
   getEditorNode: function getEditorNode() {
-    return _reactDom2.default.findDOMNode(this.editor);
+    return this.editor;
   },
 
   /*
