diff --git a/node_modules/react-quill/lib/component.js b/node_modules/react-quill/lib/component.js
index 47c34b1..c384813 100644
--- a/node_modules/react-quill/lib/component.js
+++ b/node_modules/react-quill/lib/component.js
@@ -194,14 +194,14 @@ var QuillComponent = /*#__PURE__*/function (_React$Component) {
     };
 
     _this.getEditor = function () {
-      return _this.quill;
+      return _this.quill; 
     };
 
     _this.getEditorNode = function () {
       return _reactDom.default.findDOMNode(_this.editor);
     };
-
-    _this.getEditorContainerNode = function () {
-      return _reactDom.default.findDOMNode(_this.editorContainer);
+    
+    _this.getEditorContainerNode = function () { 
+      return _this.editorContainer.current;
     };
 
     _this.getSelection = function () {
diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 10842a1..878c946 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -216,7 +216,7 @@
     };
 
     _this.getEditorNode = function () {
-      return _reactDom.default.findDOMNode(_this.editor);
+      return _this.editor.current;
     };
 
     _this.focus = function () {
